tests/test_serialize.py

```diff
@@ -85,3 +85,44 @@ class MyModel(BaseModel):
     assert m.x == 1000
     assert m.model_dump(mode='json') == {'x': '1,000'}
     assert m.model_dump_json() == b'{"x":"1,000"}'
+
+
+def test_serialize_signatures():
+    def ser_plain(v: Any, info: SerializationInfo) -> Any:
+        return f'{v:,}'
+
+    def ser_wrap(v: Any, nxt: SerializeWrapHandler, info: SerializationInfo) -> Any:
+        return f'{nxt(v):,}'
+
+    class MyModel(BaseModel):
+        f1: int
+        f2: int
+        f3: int
+        f4: int
+
+        @serializer('f1')
+        def ser_f1(self, v: Any, info: FieldSerializationInfo) -> Any:
```

Comment:
Maybe I was confused by I thought we were changing the signature to `ser_f1(self, info: FieldSerializationInfo)` - e.g. not pass the value?

Reply:
I ended up opting not to. It makes it much harder to have the following work:

```python
@serilizer('x', 'y')
def ser_xy(self, v: Any, info: FieldSerializationInfo) -> Any:
    lang = self.lang
    return internationalize(v, lang)
````

The main tradeoff is that even if you don't want to use the value you still have to accept it as a parameter.

It also made the signature introspection a bit easier because we can use the number of arguments as a conditional.